### DISCUSSION TITLE
Silent Printing Feature

### DISCUSSION THREAD

**Discussion by r3df0chs** [SafeExamBrowser/seb-win-refactoring#585] [Q&A]
Hey,

i want to print something automatically from within the SaveExamBrowser without any Dialog. 
Firefox supports "print.always_print_silent" as config in the about:config

As i read other browser does also support this feature, is there an way to do this in SEB?

Best regards

**Comment by dbuechel** [comment]
At the moment, there is no such configuration option available. However, it might be that you can force this via JavaScript, but you'd need to research that on your own. If there happens to be an option in CEF / CefSharp, then we could add that for an upcoming version.
